src/explorer.ts

```diff
@@ -1,13 +1,42 @@
 import ReactDOM = require("react-dom");
-
+import {OAuth2AuthCodePKCE} from "@bity/oauth2-auth-code-pkce"
 import Explorer = require("./components/explorer");
+import constants = require("./constants");
+
 
 /**
  * Creates and renders the API Explorer component.
  */
+
+const oauth = new OAuth2AuthCodePKCE({
+  authorizationUrl: "https://app.asana.com/-/oauth_authorize",
+  tokenUrl: "https://ccbv8pweoe.execute-api.us-east-1.amazonaws.com/default/api_explorer_oauth_beta",
+  clientId: constants.CLIENT_ID,
```

Comment:
Ensure the PROD CLIENT_ID is 38682966449842

Reply:
Checked that is the case!